04/app.js

```diff
@@ -1,9 +1,32 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+  firstName: "Adam",
+  lastName: "Nowak",
+  born: {
+    day: "14",
+    month: "04",
+    year: "1985",
+  },
+};
+
+function getDate() {
+  const date = new Date();
+  const dateToLocal = date.toLocaleDateString();
+
+  const splitDate = dateToLocal.split(".");
+  const userBirth = user.born;
+
+  const arrSplit = [];
+  const arrUser = [];
+
+  for (const key in splitDate) {
+    arrSplit.push(splitDate[key]);
+  }
+  for (const key in userBirth) {
+    arrUser.push(userBirth[key]);
+  }
+
+  let checkIfMatch = arrUser.every((i) => arrSplit.includes(i));
+  console.log(checkIfMatch);
```

Comment:
Zdecydowanie przekombinowane ;P 
Jeśli coś wymaga zbyto dużego nakładu pracy to zazwyczaj oznacza, że nie jest to najprostsza droga - trzeba szukać innego rozwiązania.

Proponuję pobrać dzień, pobrać miesiąc (i dodać 1) oraz porównać pamiętając o tym, że trzeba dane od użytkownika zamienicć na liczbę ;)